camptix.php

```diff
@@ -2933,23 +2923,26 @@ public static function esc_csv( $fields ) {
 		);
 
 		foreach( $fields as $index => $field ) {
-			// Escape trigger characters at the start of a new field
-			$first_cell_character = mb_substr( $field, 0, 1 );
-			$is_trigger_character = in_array( $first_cell_character, $active_content_triggers, true );
-			$is_delimiter         = in_array( $first_cell_character, $delimiters,              true );
 
-			if ( $is_trigger_character || $is_delimiter ) {
-				$field = "'" . $field;
+			if ( ! is_string( $field ) ) {
+				continue;
```

Comment:
What are some examples of where the input wouldn't be a string? If we do encounter that, would it be safer to do something like `$fields[ $index ] = '';` before `continue`ing?

Reply:
so the fields could be numeric. In which case fputcsv will correctly convert them to strings before exporting, and we won't need to do anything there.

Reply:
Ah, good point. In that case, would it be better to use 

```php
if ( is_numeric( $field ) ) {
	continue;
}
```

?

That way the code is more self-documenting, but unexpected things like objects would still throw an error, to the the dev know they're passing something invalid. I'm mostly just worried about some specially crafted input being able to bypass the escaping. Since this is a security function, I think it's best to err on the side of caution.

Reply:
Good idea. Done.